test/parallel/test-fs-realpath-pipe.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+// Skip Windows as /dev/stdin is not available
+if (process.platform === 'win32')
```

Comment:
Use [`common.isWindows`](https://github.com/nodejs/node/blob/master/test/common/README.md#iswindows) [`common.skip()`](https://github.com/nodejs/node/blob/master/test/common/README.md#skipmsg) here instead. See [here](https://github.com/nodejs/node/blob/415098f4997c014b4f9b7b33fc0c7f9588e1e090/test/parallel/test-cluster-disconnect-race.js#L11-L14) for an example.